src/Simulator/IncrementalRedstoneSimulator/RedstoneHandler.h

```diff
@@ -0,0 +1,127 @@
+
+#pragma once
+
+#include "World.h"
+#include "Vector3.h"
+
+
+
+
+
+class cRedstoneHandler
+{
+public:
+
+	cRedstoneHandler(cWorld & a_World) :
+		m_World(a_World)
+	{
+	}
+
+public:
+
+	// Disable the copy constructor and assignment operator
+	DISALLOW_COPY_AND_ASSIGN(cRedstoneHandler);
```

Comment:
Why not use deleted constructors? DISALLOW_COPY_AND_ASSIGN is an obsolete macro form before c++11.